service/node.go

```diff
@@ -720,31 +717,17 @@ func (s *service) NodeGetInfo(
 	// Fetch SDC GUID
 	if s.opts.SdcGUID == "" {
 		if err := s.nodeProbe(ctx); err != nil {
-			return nil, err
+			Log.Infof("failed to probe node: %s", err)
```

Comment:
**question:** Why are we no longer erroring on this situation? If this is by design due to the SDC check, can we improve the logging?

Reply:
nodeProbe does log that no SDC was found. we can propagate that error back for readability, I will look into it.